### PR TITLE
Add Innovation Projects to main nav [fix #13994]

### LINKED ISSUE #13994: Add Mozilla Innovation Projects to global nav
## Description

In the global navbar under Innovations, we should add a link to innovation projects website.

Title: Mozilla Innovation Projects
Description: Discover ways to bring bright ideas to life.
Icon: https://drive.google.com/file/d/1vROpnN5JyhLdrkBrVjn-fbpD3lnangOy/
URL: https://future.mozilla.org

Ideally to go live on Monday, 11 Dec

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds a link to future.mozilla.org to the main nav under Innovations. They're new strings so it should only be visible if those strings are translated.

## Issue / Bugzilla link
#13994

### CODE REVIEW

**Comment by Joojoo4** [comment]
Are these just emails or am I being hacked

File: bedrock/base/templates/includes/protocol/navigation/menus/innovation.html [hunk 1]
```diff
@@ -6,13 +6,19 @@
 
 {% set utm_params = 'utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=nav&utm_content=innovation' %}
 
+{% if ftl_has_messages('navigation-v2-mozilla-innovation-projects', 'navigation-v2-discover-ways-to-bring') %}
+  {% set row_count = 'mzp-l-rows-three' %}
+{% else %}
+  {% set row_count = 'mzp-l-rows-two' %}
```
Comment (reemhamz) [inline bedrock/base/templates/includes/protocol/navigation/menus/innovation.html:12]: **Praise:** smart way to set rows here!

File: media/img/nav/icons/icon-innovation-projects.svg
Comment (reemhamz) [inline media/img/nav/icons/icon-innovation-projects.svg:1]: **Suggestion(non-blocking):** I would just run `svgo` for this image. It's pretty small as is, but good to keep it as minified as possible.

**Review by reemhamz — APPROVED** [review]
Yay! I love the new icon. \o/
Just have one suggestion, but it's non-blocking.
r+wc 💡

**Pull request merged**